src/OrbitQt/orbitmainwindow.cpp

```diff
@@ -1057,6 +1058,16 @@ constexpr uint64_t kMemoryWarningThresholdKbDefaultValue = 1024 * 1024 * 8;  //
 
 void OrbitMainWindow::LoadCaptureOptionsIntoApp() {
   QSettings settings;
+  if (settings.value(kEnableSamplingSettingKey, true).toBool()) {
+    bool ok = false;
+    double sampling_period_ms = settings.value(kSamplingPeriodMsSettingKey, 1.0).toDouble(&ok);
+    CHECK(ok);
+    CHECK(sampling_period_ms > 0.0);
```

Comment:
I don't think we should CHECK here. If a user messes with the registry or the ini file we crash. I think reporting an error in the log and fall back to some default value is better in this case.

Reply:
Oh right. Added an `ERROR`, reset and fell back to the default value.